app/views/stats/index.html.erb

```diff
@@ -7,10 +7,15 @@
     }
   ]
 } %>
-<h2>Deployments per month</h2>
+
+<%= render "govuk_publishing_components/components/title", {
+  title: "Deployments per month"
+} %>
```

Comment:
This is probably a difference between live and integration, but I noticed that the page title appears slightly wrong on the stats page. It should be `Deployment stats | GOV.UK Release` but it's coming out as `Deployment stats | GOV.UK GOV.UK Release` (also I can't see what's changed that could have caused this).

![Screenshot 2019-12-02 at 10 35 59](https://user-images.githubusercontent.com/861310/69952737-bbe04d80-14ef-11ea-9b4f-a18a984785a1.png)

Reply:
Fixed. Turns out the admin layout component uses the `browser_title` and `product_name` that you pass it to [generate the title](https://github.com/alphagov/govuk_publishing_components/blob/master/app/views/govuk_publishing_components/components/_layout_for_admin.html.erb#L7). I'd set the product name to "GOV.UK Release" which was leading to a duplicate "GOV.UK".